packages/edit-post/src/components/layout/style.scss

```diff
@@ -1,3 +1,9 @@
+// This min-height is there to ensure the editing canvas extends at least to the full height available.
+// This ensures a theme that uses a background color other than white doesn't look cropped without content.
+.edit-post-visual-editor {
```

Comment:
Should we move this to the stylesheet specific for that component https://github.com/WordPress/gutenberg/blob/master/packages/edit-post/src/components/visual-editor/style.scss#L1

Reply:
Done, thanks.